terraform/projects/web-platform-tests-live/terraform.tfvars

```diff
@@ -0,0 +1,6 @@
+# Health Checks
+# this was created manually in aws sns console because
+# it cannot be done via terraform. it sends messages to
+# infrastructure@bocoup.com
+# TODO make a new health check that is more generic
+alert_sns_arn = "arn:aws:sns:us-east-1:682416359150:domainsHealthCheck"
```

Comment:
Mike I suspect we will want to create a new sns that delivers to a different email address.

I also looked into terraform to see if they recently added support for creating sns. It turns out ansible supports creating sns topics but you still can not create one that sends email via terraform. https://www.terraform.io/docs/providers/aws/r/sns_topic_subscription.html#protocols-supported

Reply:
The goal of Terraform is to have the configuration in text, but given that it's not possible, we may as well start off with a new topic whose subscriptions we can modify later. I just created arn:aws:sns:us-east-1:682416359150:web-platform-domains-health-check for this. It goes to `infrastructure+web-platform@bocoup.com` at the moment.

Reply:
Thanks @jugglinmike

Reply:
Also: I have doubts about whether the indirection of a variable is helpful here. I could be convinced that it is, though--what do you think?